config/environment.js

```diff
@@ -197,7 +197,7 @@ module.exports = function(environment) {
         support: {
             preregUrl: 'https://cos.io/prereg/',
             statusPageUrl: 'https://status.cos.io',
-            faqPageUrl: 'https://openscience.zendesk.com/hc/en-us/articles/360019737894',
+            faqPageUrl: 'https://osf-support.helpscoutdocs.com/',
```

Comment:
should this be `help.osf.io`?

Reply:
Yea, osf-support.helpscoutdocs will forward users to help.osf.io, so best to just take them to the source